src/main/java/com/shaft/tools/io/PdfFileManager.java

```diff
@@ -0,0 +1,173 @@
+package com.shaft.tools.io;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.pdfbox.cos.COSDocument;
+import org.apache.pdfbox.io.RandomAccessBufferedFileInputStream;
+import org.apache.pdfbox.pdfparser.PDFParser;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.text.PDFTextStripper;
+import org.testng.Assert;
+
+public class PdfFileManager {
+
+	private RandomAccessBufferedFileInputStream stream = null;
+	private PDFParser parser = null;
+	private COSDocument cosDoc = null;
+	private PDFTextStripper strip = null;
+	private File file;
+
+	/**
+	 * Validates the downloaded pdf document from the site under test
+	 * 
+	 * @param url                the path of the downloaded pdf file (usually will
+	 *                           be target/downloadedFiles/your-file.pdf), usually
+	 *                           will be in "target/downloadedFiles/" for the
+	 *                           project
+	 * @param milliSecondsToWait the time in milliseconds needed to wait on the
+	 *                           document to be downloaded successfully to the
+	 *                           target folder path
+	 */
+	public PdfFileManager(String url, int milliSecondsToWait) {
```

Comment:
Usually timeouts are added to the property files and not passed by the user, and it's unified across the engine that timeouts are in seconds, so we can internally convert the number in the property file from seconds to milliseconds